test/asset/image-transformation.js

```diff
@@ -89,7 +90,7 @@ test('Invalid URL: multiple parameter testing', function(assert) {
         auto: 'webp',
         format: 'jpg'
     }
-    const URL = Asset.get('url') + '?';
+    const URL = Asset['url'] + '?';
```

Comment:
@nadeem-cs @abhinav-from-contentstack  can you please confirm .get function were not working properly before ?

Reply:
@ishaileshmishra .get function is not required now. Value can be directly accessed using Object key. 
cc @abhinav-from-contentstack